### PR TITLE
Adds a "quiet" option to silence output to stdout/stderr

### PR DESCRIPTION
**Pull request by ianseyer**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func root() *cobra.Command {
 	rootCommand.Flags().StringVarP(&opts.GoTemplate, "template", "t", slice.DefaultTemplateName, "go template used to generate the file name when creating the resource files in the output directory")
 	rootCommand.Flags().BoolVar(&opts.DryRun, "dry-run", false, "if true, no files are created, but the potentially generated files will be printed as the command output")
 	rootCommand.Flags().BoolVar(&opts.DebugMode, "debug", false, "enable debug mode")
+	rootCommand.Flags().BoolVar(&opts.Quiet, "quiet", "q", false, "if true, no output is written to stdout/err")
```
Comment (patrickdappollonio) [inline app.go:74]: This should be:

```suggestion
	rootCommand.Flags().BoolVarP(&opts.Quiet, "quiet", "q", false, "if true, no output is written to stdout/err")
```

The `BoolVar` doesn't support shorthand.

File: slice/execute.go [hunk 2]
```diff
@@ -199,7 +199,9 @@ func (s *Split) store() error {
 		fmt.Fprintf(os.Stderr, "%d %s parsed to stdout.\n", s.fileCount, pluralize("file", s.fileCount))
 
 	default:
-		fmt.Fprintf(os.Stderr, "%d %s generated.\n", s.fileCount, pluralize("file", s.fileCount))
+		if !s.Quiet {
```
Comment (patrickdappollonio) [inline slice/execute.go:202]:
```suggestion
		if !s.opts.Quiet {
```

Options are propagated through `s.opts`.

**Review by patrickdappollonio — CHANGES_REQUESTED** [review]

**Comment by ianseyer** [comment]
Made the changes - how should I retrigger the builds?

**Comment by patrickdappollonio** [comment]
Sorry, first time contributors require a manual approval. I went ahead and triggered it. If successful, I'll merge.

**Pull request merged**